src/test/resources/config/sqlserver/w_perf_stresstest.yaml

```diff
@@ -0,0 +1,20 @@
+# Description: Example for custom concurrent performance stresstest workload
```

Comment:
Could we run this small test (or something with `ConcurrentPerfStresstestTaskExecutor`) in Spark CI? Since we do not have SQLServer in CI, otherwise this would not run right now?

Reply:
Yep, I have that on my todo, just wanted to create the PR so that Lluis & Jose can take a look. Keeping this comment open as a reminder.

Reply:
Added, also left a todo to migrate this to SQL Server down the line.